tests/test_functorKeys.py

```diff
@@ -31,11 +31,13 @@
 import lsst.afw.image
 import lsst.afw.table
 
-np.random.seed(500)
-
 
 class FunctorKeyTestCase(lsst.shapelet.tests.ShapeletTestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        np.random.seed(500)
```

Comment:
While this preserves the old behavior, it might be better to move this to `setUp` instead so we get deterministic behavior for test methods individually.  That's especially true if the order in which test methods are executed might not be preserved when we switch to pytest.